### ISSUE TITLE
Bolt 5 no longer supports /page/home for homepage

### ISSUE THREAD

**Issue by michaelborn** [bolt/core#2906]
Bolt 5 throws an error when attempting to render the homepage configured with `homepage: pages/home`.

## Details

* Bolt Version: 5.3.10
* Install type: Docker compose courtesy of https://github.com/bolt/project/
* PHP version: 8.0.12
* Web server: Nginx

## Bug summary

The [Bolt documentation for `config.yaml`](https://docs.boltcms.io/5.0/configuration/settings#homepage) still states the homepage "can either be a singleton like homepage, a specific record (like pages/1) or a listing of records (like entries)."

This no longer seems to be the case - Bolt 5 only supports homepage rendering as a singleton, is that correct? I'm logging this bug because I don't think the intent is to prevent rendering a `/pages/1` record... else I would have seen something in either the docs or GitHub Issues to note the change.

## Specifics

Here's my homepage config:

```yml
homepage: /page/home
homepage_template: index.twig
```

And here's the error I get when I hit the homepage route:

![Screenshot from 2021-11-04 07-11-33](https://user-images.githubusercontent.com/8106227/140304025-9f4bfba4-4f5e-40f6-8a91-1f3207aea887.png)

The issue seems to be with this commit: https://github.com/bolt/core/commit/7dfa997e63164afb2725a5008e33ec798e2356b1#diff-04640429b28b1ce688ce163c2ac3b6312edac487805c19660e707380fe1b0855

### Steps to reproduce

* Configure a homepage with `homepage: pages/1` as the docs say
* Visit the homepage route

**Comment by bobdenotter** [comment]
That feature should be supported. I think the correct syntax should be `homepage: pages/home`

It expects a `contentType/slug` combination, not a URL. Let me know if that works as expected!

**Comment by michaelborn** [comment]
Thanks for that, Bob. I think my 6am brain was confused by the zero-based array index.

I did get this working, with that pointer, and the following caveats:

1. The content type needs a `slug` defined. This seems to be the biggest change, as the content type definition name no longer "just works". We should definitely put a note in the Bolt 5 upgrade docs on this - I think I ran into it during the conimex import, as well.
2. I had to remove the leading slash, so instead of `/pages/home` I set this as `pages/home`.

Thanks for your help, Bob!
